packages/react-components/src/icons/src/EmbeddedIcon.tsx

```diff
@@ -0,0 +1,38 @@
+import { Children, ReactElement, ReactNode } from "react";
+import { augmentElement, createSizeAdapter } from "../../shared";
+
+export interface EmbeddedIconProps {
+    /**
```

Comment:
I added some documentation here, even if it's not used by storybook. It's now available through intellisense